nowplaying/cli.py

```diff
@@ -16,7 +16,10 @@
 import urllib.request
 
 import input
-import track
+import input.handler
```

Comment:
* What's the reason here for not simply having `input` and `track` available?
* Why is `import track` being removed, but `import input` will be left in place?

Reply:
This PR aims at "just" getting the code to execute again. This means there are still some rough edges that will be taken care of in later PRs.

> What's the reason here for not simply having input and track available?

We only need some like the `track.observer` parts of `track` so it makes no sense to include them at this point. The reasoning for `input` is similar.

> Why is import track being removed, but import input will be left in place?

I implemented tests for the `track` bit but didn't do so to the same effect for the `input` bits. This leads to a situation where the track parts are already a bit more consolidated than the input module.

Reply:
> > Why is import track being removed, but import input will be left in place?
> 
> I implemented tests for the `track` bit but didn't do so to the same effect for the `input` bits. This leads to a situation where the track parts are already a bit more consolidated than the input module.

Right now it gets imported multiple times, which doesn't make sense at all. So either handle it the same way as you did with `track` and only import the specific subpart or import the whole `input` as it was initially.

Reply:
> Right now it gets imported multiple times

Nope, `import input` does not also `import input.handler` as far as I can tell.

Reply:
> > Right now it gets imported multiple times
> 
> Nope, `import input` does not also `import input.handler` as far as I can tell.

Regarding `track` you previously wrote:
>>>  so it makes no sense to include all of track at this point.

I don't understand why it works for `track` but not for `input` then...

Reply:
The `input` parts aren't tested yet so I touched much less of those, which is the reason for this slight asymmetry. Testing the input parts is more work than the track stuff since there is a need for much more mocking. All this PR really aims at getting to is to somehow get the code to execute on python 3, especially parts that are extremely linked to legacy infrastructure are much less polished at this state (this affects most of the input subsystem). 

Please keep in mind that this will be refactored into something much more consistent and sane very soon (outside of the scope of this pr). In the config-abstraction/ABC PR that I'm preparing input and track will be much in line again as I refactor cli.py.

Reply:
See https://github.com/radiorabe/nowplaying/blob/c17f85ff1d1db58d855d1fa3eb51ea0acfe93f6a/nowplaying/daemon.py#L7-L10 for how I'm addressing this nitpick.